app/core/Tracker.coffee

```diff
@@ -19,7 +19,7 @@ module.exports = class Tracker extends CocoClass
     @supermodel = new SuperModel()
     @identify() # Needs supermodel to exist first
     @updateRole() if me.get('role') and not me.isSmokeTestUser()
-    if me.isTeacher() and @isProduction and not application.testing and not me.isSmokeTestUser()
+    if me.isTeacher(true) and @isProduction and not application.testing and not me.isSmokeTestUser()
       @updateIntercomRegularly()
```

Comment:
I think these setup things in Tracker will not start until the `possible teacher` refreshes the page after getting marked "possible teacher". Up to you if that's acceptable.

Reply:
@phoenixeliot Good point, I think we do want to `updateIntercomRegularly` even without a page reload after we `updateRole to `"possible teacher"`. Not exactly sure what info `updateIntercomRegularly` is sending to Intercom, but sounds useful. Do you think you could help me by adjusting this to also call `updateIntercomRegularly` when we become a teacher here?